OData/test/E2ETest/WebStack.QA.Test.OData/AlternateKeys/AlternateKeysTest.cs

```diff
@@ -0,0 +1,334 @@
+                    <PropertyValue Property=""Alias"" String=""Country"" />
+                    <PropertyValue Property=""Name"" PropertyPath=""Country"" />
+                  </Record>
+                  <Record Type=""OData.Community.Keys.V1.PropertyRef"">
+                    <PropertyValue Property=""Alias"" String=""Passport"" />
+                    <PropertyValue Property=""Name"" PropertyPath=""Passport"" />
+                  </Record>
+                </Collection>
+              </PropertyValue>
+            </Record>
+          </Collection>
+        </Annotation>
+      </EntityType>
+      <ComplexType Name=""Address"">
+        <Property Name=""Street"" Type=""Edm.String"" />
+        <Property Name=""City"" Type=""Edm.String"" />
+      </ComplexType>
+      <EntityType Name=""Company"">
+        <Key>
+          <PropertyRef Name=""ID"" />
+        </Key>
+        <Property Name=""ID"" Type=""Edm.Int32"" />
+        <Property Name=""Location"" Type=""NS.Address"" />
+        <Annotation Term=""OData.Community.Keys.V1.AlternateKeys"">
+          <Collection>
+            <Record Type=""OData.Community.Keys.V1.AlternateKey"">
+              <PropertyValue Property=""Key"">
+                <Collection>
+                  <Record Type=""OData.Community.Keys.V1.PropertyRef"">
+                    <PropertyValue Property=""Alias"" String=""City"" />
+                    <PropertyValue Property=""Name"" PropertyPath=""City"" />
+                  </Record>
+                  <Record Type=""OData.Community.Keys.V1.PropertyRef"">
+                    <PropertyValue Property=""Alias"" String=""Street"" />
+                    <PropertyValue Property=""Name"" PropertyPath=""Street"" />
+                  </Record>
+                </Collection>
+              </PropertyValue>
+            </Record>
+          </Collection>
+        </Annotation>
+      </EntityType>
+      <EntityContainer Name=""Default"">
+        <EntitySet Name=""Customers"" EntityType=""NS.Customer"" />
+        <EntitySet Name=""Orders"" EntityType=""NS.Order"" />
+        <EntitySet Name=""People"" EntityType=""NS.Person"" />
+        <EntitySet Name=""Companies"" EntityType=""NS.Company"" />
+      </EntityContainer>
+    </Schema>
+  </edmx:DataServices>
+</edmx:Edmx>";
+            var requestUri = string.Format("{0}/odata/$metadata", this.BaseAddress);
+            HttpResponseMessage response = await Client.GetAsync(requestUri);
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            string responseContent = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(expect, responseContent);
+        }
+
+        [Fact]
+        public async Task QueryEntityWithSingleAlternateKeysWorks()
+        {
+            // query with alternate keys
+            const string expect = @"{
+  ""@odata.context"":""http://sam-xu-odata:9001/odata/$metadata#Edm.String"",""value"":""special-SSN""
```

Comment:
Need to remove the odata.context which will cause it to fail on another machine during E2e test runs.